File: BookStoreAngular/src/app/Models/bookInfo.ts
export interface IBookInfo {
  id: number;
  bookName:string;
  writerName:string;
  coverURL:string;
  price:any;

}

export class BookInfo implements IBookInfo {
  id: number;
  bookName:string;
  writerName:string;
  coverURL:string;
  price:any;
}

File: BookStoreAPI/Data/Entities/BookInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace dotnetClaimAuthorization.Data.Entities
{
    public class BookInfo
    {
        public int Id { get; set; }
        public string BookName { get; set; }
        public string WriterName { get; set; }
        public string CoverURL { get; set; }
        public float Price { get; set; }
    }
}
